src/OlStyleParser.spec.ts

```diff
@@ -368,7 +368,7 @@ describe('OlStyleParser implements StyleParser', () => {
         const fontFamily = [
           ['arial', 'sans-serif'],
           ['Georgia', 'serif'],
-          ['"Neue Helvetica"', 'Helvetica', 'sans-serif']
+          ['Neue Helvetica', 'Helvetica', 'sans-serif']
```

Comment:
Are you sure the double quotes should be removed here?

Reply:
good catch, this has been fixed just now